docs/admin_guide.md

```diff
@@ -243,15 +243,15 @@ sudo nano /etc/redis/redis.conf
 # Change no to systemd, considering Ubuntu is using systemd
 ```
 
-Save and exit (ctrl+x) the file.
+Save and exit (CTRL+x) the file.
```

Comment:
If you consider "ctrl" to be wrong, then so is "CTRL". It's not an acronym, it's the word "Control", so use what most keyboards use: "Ctrl"